containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip.mdx

```diff
@@ -0,0 +1,80 @@
+---
+meta:
+  title: Moving Kubernetes nodes to routed IPs
+  description: Safely moving Kubernetes nodes to routed IPs
+content:
+  h1: Moving Kubernetes nodes to routed IPs
+  paragraph: Safely Migrating Kapsule clusters to routed IPs
```

Comment:
Don't description and paragraph usually have the same text?

Reply:
```suggestion
  paragraph: Safely moving Kubernetes nodes to routed IPs
```